### PR TITLE
WT-5014 Migrate jenkins wiredtiger perf long job to evergreen

### PR DESCRIPTION
**Pull request by Siddhartha8899**

### CODE REVIEW

File: bench/wtperf/wtperf_run_py/wtperf_run.py [hunk 1]
```diff
@@ -193,15 +196,34 @@ def setup_perf_stats():
...
+                                        output_label='Latency(insert) Max',
+                                        ops = ['insert'],
+                                        num_max = 1))
+    perf_stats.add_stat(PerfStatLatency(short_label="max_latency_read_update",
+                                        stat_file='monitor.json',
+                                        output_label='Latency(read, update) Max',
+                                        ops = ['read', 'update'],
+                                        num_max = 1))
+    perf_stats.add_stat(PerfStatMax(short_label="max_read_throughput",
+                                    pattern=r'updates,',
```
Comment (ajmorton) [inline bench/wtperf/wtperf_run_py/wtperf_run.py:220]: Nitpick: Change this pattern to `r'reads,'`. Same for `min_read_throughput` below

File: bench/wtperf/wtperf_run_py/wtperf_run.py [hunk 2]
```diff
@@ -124,15 +121,17 @@ def detailed_perf_stats(config: WTPerfConfig, perf_stats: PerfStatCollection):
...
+        run_test(config=config, test_run=test_run, index=index, arguments=arguments)
         print("Completed test {}".format(test_run))
 
 
-def run_test(config: WTPerfConfig, test_run: int, operations: List[str] = None, arguments: List[str] = None):
-    test_home = create_test_home_path(home=config.home_dir, test_run=test_run, operations=operations)
+def run_test(config: WTPerfConfig, test_run: int, index: int = 0, arguments: List[str] = None):
+    test_home = create_test_home_path(home=config.home_dir, test_run=test_run, index=index)
+    if config.verbose:
+        print("Home directory created: {}".format(test_home))
```
Comment (jeremythorp) [inline bench/wtperf/wtperf_run_py/wtperf_run.py:134]: I think there's a chance  `Home directory created` might be cause someone to think that code is creating the directory. Perhaps changing to something like `Home directory path created`  would help.

**Review by jeremythorp — APPROVED** [review]
LGTM, with one query about the "Home directory created" string in a print call.

**Review by ajmorton — APPROVED** [review]
LGTM!

File: bench/wtperf/wtperf_run_py/wtperf_run.py (hunk 1 above)
Comment (Siddhartha8899) [reply bench/wtperf/wtperf_run_py/wtperf_run.py:220]: I did that before, but it also picked up `threads, ` might have to some other way, perhaps `r'\d+ reads,'`

Comment (ajmorton) [reply bench/wtperf/wtperf_run_py/wtperf_run.py:220]: No worries then, happy to keep `updates,`

File: bench/wtperf/wtperf_run_py/wtperf_run.py [hunk 3]
```diff
@@ -43,11 +43,8 @@
 from wtperf_config import WTPerfConfig
 
 
-def create_test_home_path(home: str, test_run: int, operations: List[str] = None):
-    home_path = "{}_{}".format(home, test_run)
-    if operations:
-        # Use the first operation name as part of the home path
-        home_path += "_{}".format(operations[0])
+def create_test_home_path(home: str, test_run: int, index:int):
```
[the remark this answers is not in the document]
Comment (ajmorton) [reply bench/wtperf/wtperf_run_py/wtperf_run.py:46]: Update: This fixes the original problem reported in the ticket, but we'll also add some additional validation to batch files.

**Pull request merged**